build/CopyFilesToStagingDir.ps1

```diff
@@ -130,6 +130,10 @@ PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.ApplicationMode
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.PushNotifications.winmd $NugetDir\lib\native
 #
 # C#/WinRT Projections
+PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
+PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
```

Comment:
>pdb [](http://example.com/codeflow?start=157&length=3)

why do we put pdb in package?

Reply:
Because we do for everything else

We should cut it here if/when when we stop adding PDBs to the Transport.
I think we do it for test infrastructure/troubleshooting reasons. I don't recall if that's also the long-term plan or symbol server or such can fill the role. Regardless, not significant here for today